src/jmh/java/org/jabref/benchmarks/Benchmarks.java

```diff
@@ -92,7 +93,8 @@ private StringWriter getOutputWriter() throws IOException {
 
     @Benchmark
     public ParserResult parse() throws IOException {
-        BibtexParser parser = new BibtexParser(Globals.prefs.getImportFormatPreferences());
+        PreferencesService preferencesService = Injector.instantiateModelOrService(PreferencesService.class);
+        BibtexParser parser = new BibtexParser(preferencesService.getImportFormatPreferences());
```

Comment:
This would require to register the ImportFormatPreferences at launch time of jabref in the afterburner. Now we have lazy loading implemented. Would probably increase the launch time, as some of the preferences objects require some more construction time.